src/ameba/rule/style/verbose_block.cr

```diff
@@ -0,0 +1,162 @@
+module Ameba::Rule::Style
+  # This rule is used to identify usage of single expression blocks with
+  # argument as a receiver, that can be collapsed into a short form.
+  #
+  # For example, this is considered invalid:
+  #
+  # ```
+  # (1..3).any? { |i| i.odd? }
+  # ```
+  #
+  # And it should be written as this:
+  #
+  # ```
+  # (1..3).any?(&.odd?)
+  # ```
+  #
+  # YAML configuration example:
+  #
+  # ```
+  # Style/VerboseBlock:
+  #   Enabled: true
+  #   ExcludeMultipleLineBlocks: true
+  #   ExcludeCallsWithBlocks: false
+  #   ExcludeOperators: false
+  #   ExcludeSetters: false
+  #   MaxLineLength: ~
+  #   MaxLength: 50 # use ~ to disable
```

Comment:
Would be great to document the params. Example:
https://github.com/crystal-ameba/ameba/blob/51b0a07e810501895a71a8f8183ccf752d10d083/src/ameba/rule/style/redundant_next.cr#L49-L89